Project 2/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--sources
+image:https://www.owame.com/products/japanese-mori-girl-long-skirts-women-high-waist-army-green-umbrella-skirts-pocket-female-cotton-midi-saia-solid-color
+fonts: cambria and gill sans
+-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <section class="clearfix">
+
+        <img src="images/Capture.JPG" alt="green A line skirt" class="left">
```

Comment:
As a frequent online shopper, I like to know critical details about a design: how it closes and if it has **pockets**  -- I bet screenreader users would benefit from those details, too, no? Especially since that info isn't in the text description, the `alt` text is a great place for blind users to get that info.